src/main/java/com/courtcanva/ccfranchise/services/FranchiseeService.java

```diff
@@ -27,12 +37,16 @@ public class FranchiseeService {
 
     private final StaffService staffService;
 
+    private final SuburbService suburbService;
+
+    private final SuburbMapper suburbMapper;
+
     @Transactional
     public FranchiseeAndStaffDto createFranchiseeAndStaff(FranchiseePostDto franchiseePostDto, StaffPostDto staffPostDto) {
 
-        if (checkFranchiseeIsExist(franchiseePostDto.getAbn())) {
+        if (ifFranchiseeExists(franchiseePostDto.getAbn())) {
```

Comment:
Sorry to mislead you, but I think `ifFranchiseeExist` is not a good method name either. use `franchiseeExists` instead. You can check the following link to see the discussion about how to name the boolean here. https://www.reddit.com/r/AskProgramming/comments/7mlcch/is_isexist_acceptable_for_a_variable_name/